src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -197,6 +211,48 @@ const HotspotDetails = ({
     )
   }, [t])
 
+  const getDistance = useCallback(
+    (otherHotspot: Hotspot) => {
+      if (
```

Comment:
Could use `!hotspotHasValidLocation(hotspot) || !hotspotHasValidLocation(otherHotspots)`

Reply:
Nice helper, but now TS is complaining below that they may be undefined. Know any way to fix that?